src/components/pages/devices/devices.js

```diff
@@ -1,14 +1,30 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 import React, { Component } from 'react';
+import { Btn } from 'components/shared';
 
 import './devices.css';
 
 export class Devices extends Component {
-  render () {
+  constructor(props) {
+    super(props);
+    if (props.initialize) {
```

Comment:
Remove the if. If initialize isn't included then something is wrong and the component should die. Check the existence with [propTypes](https://reactjs.org/docs/typechecking-with-proptypes.html)

Reply:
I added it to make the test pass.  The component handles when the device list isn't passed, but blows up without the handler for the refresh button... so I made it more resilient to that.  
Alternatively, I can change the test.

Reply:
The fix there would be to fake the prop in the test.